docs/userguide/main.md

```diff
@@ -1036,6 +1036,40 @@ spec:
   description: Sample tenant
 ```
 
+
+### PodDisruptionBudget
+
+
+The PDB (Pod Disruption Budget) is a Kubernetes resource that helps ensure the high availability of applications by defining the acceptable disruption level during maintenance or unexpected events.
+It specifies the minimum number of pods that must remain available to maintain the desired level of service.
+The PDB definition is unique for every nodePool.
+Please be aware that you must provide `MinAvailable` and `MaxUnavailable` configurations in order to use PDB feature.
+
+
+```yaml
+apiVersion: opensearch.opster.io/v1
+kind: OpenSearchCluster
+...
+spec:
+   nodePools:
+      - component: masters
+        replicas: 3
+        diskSize: "30Gi"
+        pdb:
+          enable: true
+          MinAvailable: 3
+          MaxUnavailable: 0 
```

Comment:
```suggestion
          minAvailable: 3
          maxUnavailable: 0 
```
Also: Example will not work with both fields being set

Reply:
Comment was not addressed. Example cannot have both fields set, that would be invalid,